packages/framework/ai-collab/README.md

```diff
@@ -1,34 +1,194 @@
 # @fluid-experimental/ai-collab
 
-Utilities for using SharedTree with LLMs.
-This package aims to assist SharedTree-based apps that want to leverage LLMs.
+## Description
+The ai-collab client library makes adding complex, human-like collaboration with LLM's built directly in your application as simple as one function call. Simply pass your SharedTree and ask AI to collaborate. For example,
+- Task Management App: "Reorder this list of tasks in order from least to highest complexity."
+- Job Board App: "Create a new job listing and add it to this job board"
+- Calender App: "Manage my calender to slot in a new 2:30 appointment"
+
+## Usage
+
+### Your SharedTree types file
+This file is where we define the types of our task management application's SharedTree data
+```ts
```

Comment:
```suggestion
### Your SharedTree types file

This file is where we define the types of our task management application's SharedTree data.

```ts
```

Reply:
Updated in commit b700ced170a403db24c887b25ce4917392948b3c